src/app.ts

```diff
@@ -3,12 +3,15 @@ const app: express.Application = express();
 import { index } from './routes/index';
 import { nexus } from './routes/nexus';
 import { scores } from './routes/scores';
+import { games } from './routes/games';
 
 app.set('view engine', 'pug');
+app.use('/static', express.static('public'));
```

Comment:
Are you still using this, or maybe did you mean for it to be 'vendor' now?

Reply:
Fixed in df2b5b1